theano/sandbox/gpuarray/tests/test_opt.py

```diff
@@ -7,6 +7,7 @@
 from theano.tensor.tests import test_basic
 
 import theano.sandbox.gpuarray
+from theano.sandbox.gpuarray import basic_ops
```

Comment:
I would rather you add the gpu_contiguous import to the ..basic_ops line below.

If you really want to import basic_op do it with a relative import `from .. import basic_ops`.